toy_settings/django_back_end/models.py

```diff
@@ -25,3 +25,14 @@ class Event(models.Model):
     timestamp = models.DateTimeField()
     payload = models.CharField(max_length=500)
     payload_converter = cattrs.preconf.json.make_converter()
+
+
+class Sequence(models.Model):
+    event = models.ForeignKey(Event, on_delete=models.PROTECT)
```

Comment:
Question: This prevents Events being deleted unless their Sequence has been manually removed first. Was that the intention?

Reply:
Events shouldn't be removed ever, so this shouldn't matter -- I guess cascade would also be fine